Assets/Scripts/MenuManager.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// Provides functions for the buttons on the title screen
+/// </summary>
+public class MenuManager : MonoBehaviour
+{
+    [SerializeField]
+    [Tooltip("The panel for the main menu.")]
+    private GameObject mainMenuPanel;
+
+    [SerializeField]
+    [Tooltip("The panel for the credits.")]
+    private GameObject creditsPanel;
+
+    [SerializeField]
+    [Tooltip("The name of the next scene to load.")]
+    private string nextScene;
+
+    /// <summary>
+    /// A button to select when changing panels
+    /// </summary>
+    private Selectable buttonToSelect;
+
+    /// <summary>
+    /// Loads the game
```

Comment:
you can leave a note that this is meant to be called from a button script in the editor, if that's the case.